### DISCUSSION TITLE
Need help with making my ip address to my subdomain on namecheap

### DISCUSSION THREAD

**Discussion by Saboreq** [rejetto/hfs#369] [General]
i changed address/domain in hfs admin settings to files.saboreq.com and made A files.saboreq.com dns record to point to my IP but this doesnt work, when im clicking check button it says domain is okay but when im trying to go to files.saboreq.com then im getting "Hmmm… can't reach this page" add me on discord please for faster fix username: saboreq

**Comment by rejetto** [comment]
your domain may not work from your local network. This doesn't mean it doesn't work for people over the internet.
If you have a mobile connection, ensure your phone is NOT connected to the wifi, then access your domain from your phone.

**Reply by Saboreq** [reply]
still doesnt work idk why ://

**Reply by PH5HP** [reply]
Hi,

I my case I cannot access localy by https, but it works for other people on the internet. Locally it works with http. So in my case it works without any problem. I refer in the DNS record to ftp.ph5hp.nl Nothing more.

Kind regards,
Henk - ftp.ph5hp.nl

**Reply by Saboreq** [reply]
How did u managed to get rid of port in url?  i can only access my site when i will type :8080 after url ;=;

**Reply by rejetto** [reply]
you have the port in the URL because you are NOT using the standard port.
the standard port for https is 443

**Reply by Saboreq** [reply]
cause i cant use 443 for some reason it says some routers might somehting blah blah under 1024

**Reply by rejetto** [reply]
then you are screwed

**Comment by rejetto** [comment]
<img width="486" alt="image" src="https://github.com/rejetto/hfs/assets/1367199/0dc6252b-4d00-482a-b329-4c9c92bcec9f">

your domain is not working, this has nothing to do with hfs

**Reply by Saboreq** [reply]
![image](https://github.com/rejetto/hfs/assets/118023809/48209635-bbd0-4db8-951a-03395987e5bb)

![image](https://github.com/rejetto/hfs/assets/118023809/b8cb6d97-8be8-4948-a177-069cd82a6d6e)

![image](https://github.com/rejetto/hfs/assets/118023809/08bc943f-0c8d-436e-86b8-bf081c432b6f)

i changed from files.saboreq.com to test.saboreq.com but still doesnt work ://

**Comment by rejetto** [comment]
you don't need to "hide" your ip. Your domain is the same as your ip. If you publish the domain, you are publishing your ip.

ok, domain is ok now.
what does "verify" on hfs says?

**Reply by Saboreq** [reply]
Im trying to host HFS on my own PC and im trying to connect domain to this.

wdym verify on hfs?

**Comment by rejetto** [comment]
in the same page where you clicked "check" for your domain, there is another link/button called "verify", below this symbol
<img width="112" alt="image" src="https://github.com/rejetto/hfs/assets/1367199/d2df6301-b523-4bff-be7e-af2e40f7692d">

**Reply by Saboreq** [reply]
![image](https://github.com/rejetto/hfs/assets/118023809/8075fd48-c91e-4d2d-b0e0-370ca0a52726)

**Comment by rejetto** [comment]
then your port is reachable but you get an "empty response".
that may be because of several things.
let me see your config.yaml file

**Reply by Saboreq** [reply]
version: 0.48.2
base_url: https://test.saboreq.com
https_port: 8080
cert: self.cert
private_key: self.key
vfs:
  children:
    - source: C:\Users\matus\Desktop\HFS\Pliki
      can_see: "*"
      can_read: "*"
      can_list: "*"
      can_upload: "*"
      can_delete: "*"
      children: []
accounts:
  saboreq:
    ignore_limits: true
    admin: true
    srp: 157103341678880742432186030640998081875684149426288923209728058223129246339455253983407880276183616044838122221462263440102623588067712860171928695093495502607558370713240437595068035977396566771578138232087977700477655631391594252979864771580093105095400385460055415118127308588984970224952143733574285885268|6254680557921614397683995063201380142073883236716176468584626887547050506159240769200591291797815028038758493747404063866040620859607545792024183234652401157454714335771363919100233027402486784571974857571074351660577207056666942143909731466909115168495680924227391390878834829887600540526562522632258509960232129085357644408885534956389075749882541897990391319123574845200716594325892605307786965074505277545111185328549763269733647336188936611894369234593095927144503872892396961389273966833775735708882613774113728598777984582552526612748741390110837823002852790500248675069535584602964228172661189070454713505050
  zevst:
    ignore_limits: true
    srp: 85333079201335287475289325453563958349347953303122504881060238904090758428817053445772406223720636326505774739687453738607284860169194000880310930006281873216190598355932574025576299747084849835328833426385492015243072489023714767827135089156014904513757712728136221152197301062220816448751893966412563950621|7172788381946109921320550462117892833616779080590494837772944209909603067050802857705695021599335927981035119334046810552651756750399999421454651925869764715086900801379877464671638223754466266467379938318922672428255559484859767123697895648411729748715696120523833888194400881121075797075067746891646631757000282642300328953461920348748180251386460801406920286178498886422658796880075503750574736142757403076677855805425320425123400953305229552906519151486059590245037841573658756196222951715277488000537430685288457068926133886570115647678131746763099125290867392895756657736840676103830237051254669483507935478081
favicon: C:\Users\matus\Downloads\icon.ico
enable_plugins:
  - antibrute
  - list-uploader
  - download-counter
  - hfs-space-gray-theme
  - simple-player
  - file-icons
plugins_config:
  file-icons:
    icons:
      - iconFile: C:\Users\matus\Downloads\icon.ico
        ext: exe
  download-counter:
    where: list
  list-uploader:
    display: user

**Comment by rejetto** [comment]
i see nothing strange here.
look into Admin > logs > console
if you can find anything useful

**Reply by Saboreq** [reply]
![image](https://github.com/rejetto/hfs/assets/118023809/2bb8da0b-37b1-415a-aa18-29181dfe20f8)
![image](https://github.com/rejetto/hfs/assets/118023809/9437b788-0d1e-4c29-bd0e-3738302b8f47)

**Comment by rejetto** [comment]
your server is working, i can access it
<img width="1099" alt="image" src="https://github.com/rejetto/hfs/assets/1367199/a35d1c05-cd16-4cd6-a6d2-e3edc07d0ecc">

**Reply by Saboreq** [reply]
im getting cant reach this page all time :/ on my pc or on my phone on which im using diffrent network connection

**Comment by rejetto** [comment]
show me your browser

**Reply by Saboreq** [reply]
nvm i just realized i forgot to put one dot -.- but earlier i couldnt connect still now i can wtf

**Comment by rejetto** [comment]
you are using a self-signed certificate.
you see the "not secure" in the browser?
my advice is that you use let's encrypt for a better certificate.
you can find it in Admin > internet > https

**Reply by Saboreq** [reply]
yeah i know but idk how to request for certificate haha i mean what exactly to put here 
![image](https://github.com/rejetto/hfs/assets/118023809/da13ae70-4a1d-45b7-9e57-ce68437dbaa2)

**Comment by rejetto** [comment]
you don't know what to put in the "certificate domain" ?

**Comment by rejetto** [comment]
would "domain for your certificate" be clearer?

**Reply by Saboreq** [reply]
yeah but i mean only test.saboreq.com or with port?

**Reply by rejetto** [reply]
without port, "domain" has never a port, that's the url.
i can put an example under the field, to make it clearer

**Comment by rejetto** [comment]
so this would be ok
<img width="332" alt="image" src="https://github.com/rejetto/hfs/assets/1367199/c74e1508-4e32-4583-98a8-f397507dbfef">

**Reply by Saboreq** [reply]
![image](https://github.com/rejetto/hfs/assets/118023809/22b04702-4d38-494f-928c-20110697fd5c)

**Reply by Saboreq** [reply]
nvm worked

**Reply by Saboreq** [reply]
Also is there possible to add disk space from another laptop with windows connected to the same wifi?

**Reply by rejetto** [reply]
well, once i successfully shared on hfs running on my mac a folder that was from my other Windows computer (using Windows file sharing), so i guess it's a yes.
But it's thanks to the operating system, not really hfs.

**Comment by rejetto** [comment]
> ![image](https://user-images.githubusercontent.com/118023809/271955665-22b04702-4d38-494f-928c-20110697fd5c.png)

do you by any chance remember what you did to get this error?

**Reply by Saboreq** [reply]
i dont remember
